### PR TITLE
Feature button position

### PR DESCRIPTION
**Pull request by bnaolu**

### CODE REVIEW

File: _layouts/product.html [hunk 1]
```diff
@@ -18,12 +18,15 @@ <h1 class="margin-top-0">{{ page.title }}</h1>
...
         {% endfor %}
     </p>
-
+   
     <div class="grid-container padding-x-0">
         <div class="grid-row">
           <div class="tablet:grid-col tablet:padding-right-1">
             <img src="{{ page.feature-image }}" alt="{{ page.feature-image-description }}">
             <button class="usa-button usa-button--unstyled margin-y-2">View more images</button>
+            <div class="margin-y-2">
```
Comment (pglevy) [inline _layouts/product.html:27]: We don't need the extra div here. You can apply the margin spacing class to the button element directly.

File: _layouts/product.html [hunk 2]
```diff
@@ -18,12 +18,15 @@ <h1 class="margin-top-0">{{ page.title }}</h1>
...
+              <button class="usa-button  usa-button--hover">Add to List</button>
```
Comment (pglevy) [inline _layouts/product.html:28]: 1. Remove usa-button--hover class. This is just for presenting documentation of UI. The hover state will occur naturally as people interact with it on a live page.
2. Let's stick with sentence case for button copy, so "Add to list".

**Review by pglevy — CHANGES_REQUESTED** [review]
Looking good, @bnaolu! Just had a couple minor things to tweak. You can work off the same branch and publish your new commits there. It will flow into this same PR.

File: _layouts/product.html (hunk 1 above)
Comment (bnaolu) [reply _layouts/product.html:27]: When I remove the div, the button becomes in-line with the "view image" link. Is that okay? 
Screen Shot 2021-05-14 at 12 51 01 PM

**Review by pglevy — APPROVED** [review]

**Comment by pglevy** [comment]
Nice work, @bnaolu!

**Pull request merged**